### DISCUSSION TITLE
Font size and window resizing

### DISCUSSION THREAD

**Discussion by diegodorado** [asciinema/discussions#74] [General]
When I change the font size in the terminal ( I use termite) the terminal adjusts correctly.
But the cast doesnt take into account those font size changes apparently.

Here's a cast that shows the problem
https://asciinema.org/a/313367

**Comment by ku1ik** [comment]
This is a tricky case. The recorder doesn't save terminal resize events in the output asciicast file. Even if it did, then the web player would need to support that in some way, and it's not quite clear how the resizing should behave given the player can be embedded on any website inside its layout (either standalone, or as iframe).
